cmd/snap-update-ns/entry_test.go

```diff
@@ -105,10 +107,25 @@ func (s *entrySuite) TestXSnapdGID(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(gid, Equals, uint64(0))
 
-	// Group is parsed from the x-snapd.gid = option.
-	nogroupGID, err := osutil.FindGid("nogroup")
-	c.Assert(err, IsNil)
-	e = &mount.Entry{Options: []string{"x-snapd.gid=nogroup"}}
+	// Group is parsed from the x-snapd-group= option.
+	var nogroup string
+	var nogroupGID uint64
+	// try to cover differences between distributions and find a suitable
+	// 'nogroup' like group, eg. Ubuntu uses 'nogroup' while Arch uses
+	// 'nobody'
+	for _, grp := range []string{"nogroup", "nobody"} {
```

Comment:
Thank you for this change, I think that's the right thing to do and if we ever find a distro which has yet another name for nobody, we can add it here.